scripts/installers/Dockerfile.ngramtools

```diff
@@ -0,0 +1,13 @@
+# How to use? Build it from NeMo root folder:
+# 1. git clone https://github.com/NVIDIA/NeMo.git && cd NeMo
+# 2. DOCKER_BUILDKIT=1 docker build -t nemo:22.12.2 -f ./scripts/installers/Dockerfile.ngramtools .
+
+from nvcr.io/nvidia/nemo:22.12
```

Comment:
Whose going to maintain and update this tag ? Is it possible to automatically find the latest tag (more Nemo does not use latest tag for containers)

Reply:
not sure if it will work with new images. May just remove Dockerfile.ngramtools

Reply:
can update it manually